app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,12 @@
 <resources>
     <string name="app_name">FOrder</string>
+    <string name="mgs_product_time_out">This product is time out, you can not order this product!</string>
     <string name="mgs_this_order">Would you like this order?</string>
     <string name="mgs_order_all">Would you like order all?</string>
     <string name="mgs_delete_product">Would you like delete this product?</string>
     <string name="action_yes">Yes</string>
     <string name="action_no">No</string>
+    <string name="action_ok">Ok</string>
```

Comment:
WARNING: [Typos] "Ok" is usually capitalized as "OK"
 This check looks through the string definitions, and if it finds any words that look like likely misspellings, they are flagged.